cloudify_cli/commands/logs.py

```diff
@@ -32,35 +32,37 @@ def logs():
 
 
 def _archive_logs(logger,
-                  host_string='',
+                  host_string,
                   key_filename=None):
     """Creates an archive of all logs found under /var/log/cloudify plus
     journalctl.
     """
-    archive_filename = 'cloudify-manager-logs_{0}_{1}.tar.gz'.format(
-        ssh.get_manager_date(),
+    archive_filename = 'cloudify-host-logs_{0}_{1}.tar.gz'.format(
+        ssh.get_host_date(host_string),
         env.profile.manager_ip
         if not host_string else host_string.split('@')[1])
     archive_path = os.path.join('/tmp', archive_filename)
     journalctl_destination_path = '/var/log/cloudify/journalctl.log'
 
-    ssh.run_command_on_manager(
+    ssh.run_command_on_host(
         'journalctl > /tmp/jctl && '
         'mv /tmp/jctl {0}'.format(journalctl_destination_path), use_sudo=True,
         host_string=host_string, key_filename=key_filename)
     logger.info('Creating logs archive in manager: {0}'.format(archive_path))
     # We skip checking if the tar executable can be found on the machine
     # knowingly. We don't want to run another ssh command just to verify
     # something that will almost never happen.
-    ssh.run_command_on_manager(
+    ssh.run_command_on_host(
         'tar -czf {0} -C /var/log cloudify '
         '-C /opt/manager cluster_statuses '
-        '--ignore-failed-read'.format(archive_path),
+        '--warning=no-file-changed'.format(archive_path),
```

Comment:
Why this change was needed?

Reply:
According to [man tar](http://man7.org/linux/man-pages/man1/tar.1.html), `--warning=no-file-changed` is used to cover the case where the file is changed while it is being read, not `--ignore-failed-read`. The only thing is that it still returns 1 as the return code, meaning it failed. Because of that, I added the following command to verify the tar was truly created. 
As a remark, while trying to run `cfy logs download` on a cluster manager node before changing the flag (i.e. while it was --ignore-failed-read) I got an error of file is changed while it is being read.